_docs/best-practices/best-practices-intro.md

```diff
@@ -0,0 +1,3 @@
+# Best Practices
+
+Here, we document some best practices for creating tools as we understand them. Our intention is that this document will evolve as descriptor languages evolve so feel free to provide suggestions and/or improvements.  
```

Comment:
This seems a little too deprecatory to me. Plus redundant to say `here, we document`... I suggest simply something like `Following are some best practices for creating tools. Our intention is that...`